src/components/event/quick-actions/quick-actions.js

```diff
@@ -0,0 +1,42 @@
+import React, { createContext } from 'react';
+import { connect } from 'react-redux';
+import { useFilterActions } from '../filter/filter-hooks';
+import { OrderEvents } from './order/order-events';
+import { JoinedEventsFilter } from './joined-events/joined-events-filter';
+import { get_events } from '../../../actions/event/event-list-action';
+import { BookmarkedEvents } from './bookmarks/bookmarked-events';
+
+export const RefreshEventsContext = createContext();
+
+const QuickActions = ({ userId, getEvents }) => {
+    const { getQueryWithRequestFilters } = useFilterActions();
+
+    const refreshEvents = () => {
+        const filterQuery = getQueryWithRequestFilters();
+        getEvents(filterQuery);
+    };
+
+    return (
+        <div className="d-flex justify-content-end">
+            <RefreshEventsContext.Provider value={refreshEvents}>
+                <OrderEvents />
+                {userId &&
+                    <>
+                        <JoinedEventsFilter />
+                        <BookmarkedEvents />
+                    </>
+                }
+            </RefreshEventsContext.Provider>
+        </div>
+    );
+};
+
```

Comment:
write proptypes and default props, do this in all component

Reply:
Done